File: csharp/08_classes.cs
using System;

class Person {
    public string Name { get; set; }
    public int Age { get; set; }

    public Person(string name, int age) {
        Name = name;
        Age = age;
    }

    public string Greet() {
        return $"hi i am {Name} and i am {Age}";
    }
}

class Program {
    static void Main() {
        var me = new Person("ulas", 20);
        Console.WriteLine(me.Greet());
        Console.WriteLine(me.Name);
    }
}

File: ruby/08_classes.rb
class Person
  attr_reader :name, :age

  def initialize(name, age)
    @name = name
    @age = age
  end

  def greet
    "hi i am #{@name} and i am #{@age}"
  end
end

me = Person.new("ulas", 20)
puts me.greet
puts me.name
